File: app/src/main/java/android/itacademy/by/dz6/student/Student.java
package android.itacademy.by.dz6.student;

public class Student {
    private String firstName;
    private String lastName;
    private String textUrl;
    private int Id;

    public Student(String firstName, String lastName, String textUrl) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.textUrl = textUrl;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setFirstName(String name) {
        this.firstName = name;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setId(int id) {
        Id = id;
    }

    public int getId() {
        return Id;
    }

    public void setTextUrl(String textUrl) {
        this.textUrl = textUrl;
    }

    public String getTextUrl() {
        return textUrl;
    }
}

File: app/src/main/java/android/itacademy/by/dz6/student/Student.kt
package android.itacademy.by.dz6.student

class Student {
    var firstName: String? = null
    var lastName: String? = null
    var textUrl: String? = null
        private set
    var id: Int = 0

    constructor(firstName: String, lastName: String, textUrl: String) {
        this.firstName = firstName
        this.lastName = lastName
        this.textUrl = textUrl
    }

    constructor(id: Int, firstName: String, lastName: String, textUrl: String) {
        this.firstName = firstName
        this.lastName = lastName
        this.textUrl = textUrl
        this.id = id
    }
}